### ISSUE TITLE
Not Working in Android 9 and 10

### ISSUE THREAD

**Issue by kennguyen1506** [AgoraIO-Community/react-native-agora#253]
I have updated to the version.
Rtc.Create, Rtc.RemoteView and Rtc. does not seem to work in android 9 and 10 on release build.

I managed to get it to work on android 7 and below, havent tested with android 8 yet.

Does anyone have a solution for this? Does it work for you on Android 9 or 10?

**Comment by LichKing-2234** [comment]
yes, it works for me on android 9, have you tried our quick start?

**Comment by kennguyen1506** [comment]
https://github.com/AgoraIO-Community/Agora-RN-Quickstart/tree/sdk-v3 
I cant because it is currently down. Have you made any update?

**Comment by LichKing-2234** [comment]
@kennguyen1506 the sdk-v3 brach has been merged to master.

**Comment by kennguyen1506** [comment]
@LichKing-2234 Im currently re-configuring my project following the quickstart. Will come back with updates within a couple days. Thank you for your help so far!

**Comment by kennguyen1506** [comment]
the quick start project definitely works, however I tried to follow your build config, run-time permissions, it still would not let the camera turn on in Android 9 or 10 with my app. I wonder other libraries I'm using is causing this issue. Will comeback with more updates soon.

**Comment by kennguyen1506** [comment]
@LichKing-2234 hi there! I successfully resolved the issues. Thank you so much. Like I was suspected, there was a location tracking scripts I used before init userId and channels. This worked well on iOS but since Android required a https permission for this to work on release build, our entire flow was halted. Just adding a permission to request https access in Manifest worked. Thank you for your help!
